storybook/package.json

```diff
@@ -8,10 +8,15 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.3.4",
+    "@nuxeo/nuxeo-elements": "^3.0.0-SNAPSHOT",
```

Comment:
Not sure what is supposed to happen, but if we want always the snapshot we should remove the `^`